Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -1320,16 +1625,44 @@ bool msblenContext::sendMaterials(bool dirty_all)
     return true;
 }
 
+void msblenContext::requestServerInitiatedMessage()
+{
+    if (m_settings.ExportSceneCache) {
+        return;
+    }
+
+    m_sender.on_server_initiated_response_received = [this](auto properties, auto entities, std::string messageFromServer) {
+        m_property_manager.updateFromServer(properties, entities);
+
+        if (messageFromServer == "sync") {
```

Comment:
Would it be better to use a const instead of hardcoding here? Maybe have a file with server messages definitions

Reply:
fixed